docs/src/concepts.md

```diff
@@ -0,0 +1,160 @@
+
+
+## Element Features
+Through our interviews and direct collaboration on the precursor projects, we identified the common motifs to create the Calcium Imaging Element with the repository hosted at https://github.com/datajoint/element-calcium-imaging.
+
+Major features of the Calcium Imaging Element include:
+
++ Calcium-imaging scanning metadata, also compatible with mesoscale imaging and multi-ROI scanning mode
++ Tables for all processing steps: motion correction, segmentation, cell spatial footprint, fluorescence trace extraction, spike inference and cell classification
++ Store/track/manage different curations of the segmentation results
++ Ingestion support for data acquired with ScanImage and Scanbox acquisition systems
++ Ingestion support for processing outputs from both Suite2p and CaImAn analysis suites
++ Sample data and complete test suite for quality assurance
+
+The processing workflow is typically performed on a per-scan basis, however, depending on the nature of the research questions, different labs may opt to perform processing/segmentation on a concatenated set of data from multiple scans. To this end, we have extended the Calcium Imaging Element and provided a design version capable of supporting a multi-scan processing scheme.
+
+
+## Element Architecture
+Each node in the following diagram represents the analysis code in the workflow and the corresponding table in the database.  Within the workflow, Element Calcium Imaging connects to upstream Elements including Lab, Animal, and Session. For more detailed documentation on each table, see the API docs for the respective schemas.
+
+The Element is composed of two main schemas, `scan` and `imaging`. To handle
+several use cases of this pipeline, we have designed two alternatives to `imaging` schemas,
+including `imaging_no_curation` and `imaging_preprocess`.
+
+
+- `imaging` module - Multiple scans are acquired during each session and each scan is
+processed independently.
+
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element.svg)
+
+- `imaging_no_curation` module - Same as `imaging` module, without the Curation table.
+
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element.svg)
+
+
+- `imaging_preprocess` module - Same as `imaging` module. Additionally, pre-processing
+steps can be performed on each scan prior to processing with Suite2p or CaImAn.
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element_preprocess.svg)
+
+
+### `lab` schema ([API docs](../api/workflow_calcium_imaging/pipeline/#workflow_calcium_imaging.pipeline.Equipment))
```

Comment:
Does this need a `#` before `#workflow`?

Reply:
It works with #.